logstash_async/database.py

```diff
@@ -29,6 +29,10 @@ class DatabaseLockedError(Exception):
     pass
 
 
+class DiskIOError(Exception):
```

Comment:
Could we name it `DatabaseDiskIOError` so it is inline with `DatabaseLockedError`?